test/snapper_dialogs_include_test.rb

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env rspec
+# coding: utf-8
+
+ENV["Y2DIR"] = File.expand_path("../../src", __FILE__)
+
+require "yast"
+
+class DummyClass < Yast::Client
+  def initialize
+    Yast.include self, "snapper/dialogs.rb"
+  end
+end
+
+describe "Yast::SnapperDialogsInclude" do
```

Comment:
no need for string, better pass object directly, it then allow to access it easier with described class and subject.